staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/apiserver.go

```diff
@@ -198,12 +201,17 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget)
 		crdClient.Apiextensions(),
 		crdHandler,
 	)
+	openapiController := openapicontroller.NewController(s.Informers.Apiextensions().InternalVersion().CustomResourceDefinitions())
```

Comment:
this is constructed even when the feature is disabled... does that mean the event handlers (addCustomResourceDefinition, etc) will fill a queue that will never be drained? should construction of this controller be protected by the feature gate?

Reply:
good point. adding the gate